exercises/practice/killer-sudoku-helper/.meta/example.ex

```diff
@@ -0,0 +1,26 @@
+defmodule KillerSudokuHelper do
+  @doc """
+  Return the possible combinations of `size` distinct numbers from 1-9 excluding `exclude` that sum up to `sum`.
+  """
+  @spec combinations(cage :: %{exclude: [integer], size: integer, sum: integer}) :: [[integer]]
+  def combinations(%{exclude: exclude, size: size, sum: sum}) do
+    numbers = [9, 8, 7, 6, 5, 4, 3, 2, 1] -- exclude
```

Comment:
I assume you removed the descending range to make it pass in older Elixirs, but I would have still kept it, just with a `Enum.sort` instead. But it doesn't matter 🙂

Reply:
Yes, I first had `9..1//-1`, and when it got rejected I wrote the full list explicitly out of spite :D